app/controllers/provider/admin/account/email_configurations_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class Provider::Admin::Account::EmailConfigurationsController < Provider::Admin::Account::BaseController
```

Comment:
Actually, is this correctly scoped? Shouldn't in be `Provider::EmailConfigurationsController` since it's only for the master provider?

Reply:
The thing is that in the future we may enable this for other providers as well.

wrt scoping - I'm not sure about this. I don't understand the convention very well. `Provider::Admin` sounds good to me because only provider admins should be ever able to manage this. `Account` - I don't know what is this about. So if you think another scope makes more sense - I'm very open to it.

Reply:
Yeah, the convention about scoping is very loose. `Provider::Admin` makes sense, we could remove the `Account` part.

I think the idea ultimately is to share the same structure between UI and controllers. So if the scope were `Provider::Admin::Account` the URL should be like `p/admin/accounts/:id/email_configuration`

Reply:
Agreed - depending on UI follow Rails conventions.